apps/pie-storybook/types/StoryOptions.ts

```diff
@@ -0,0 +1,20 @@
+type BackgroundColor = 'light (container-default)' |
+    'dark (container-dark)' |
+    'background-subtle' |
+    'brand orange' |
+    'saddlebrown' |
+    'aquamarine';
+
+type BackgroundValue = {
+    name: BackgroundColor,
+    value: string,
```

Comment:
is it worth union typing the values?

Reply:
I did briefly consider this, but I'm pretty sure they're only used in that one object so I'm not sure there's much value in it, what do you think?
I did just notice one of the imports is wrong in another file so lemme fix that.

Reply:
It seems a bit odd that we could add a key of `saddlebrown` with a value of `container-default` if that makes sense?

Reply:
We paired on this and couldn't quite find a nice improvement, so we'll leave this for now.